### ISSUE TITLE
Composer dependencies not "stable"

### ISSUE THREAD

**Issue by darylteo** [octobercms/october#2539]
##### Expected behavior
composer require october/october dev-master should work

##### Actual behavior
  Problem 1
    - Installation request for october/october dev-master -> satisfiable by october/october[dev-master].
    - october/october dev-master requires wikimedia/composer-merge-plugin dev-master -> satisfiable by wikimedia/composer-merge-plugin[dev-master] but these conflict with your requirements or minimum-stability.

##### Reproduce steps
composer require october/october dev-master

##### October build
Not sure.

##### Workarounds

 - add minimum-stability to composer.json to "dev"

##### Ideal solutions

 - up dependency to latest stable
 - up release to 1.0.0 for october/october on packagist.

**Comment by daftspunk** [comment]
This is a duplicate of #2256, which we have no viable solution to at this stage. The workaround is the solution at the moment.

**Comment by darylteo** [comment]
Apologies. Search didn't reveal anything when I tried looking.

Are we waiting on some dev feature of the chunk merge plugin to be released?

I'm really looking forward to working with OctoberCMS for projects :)

**Comment by daftspunk** [comment]
October uses builds instead of versions, deployed as rolling updates. Technically [October is 1.0.387](http://octobercms.com/changelog) but composer doesn't support this approach very well, since it relies on tags in git to be created.

**Comment by darylteo** [comment]
Thanks @daftspunk for the information.

After digging in the historical info a bit more it appears that I am not the first person to look for   #2413. May I suggest that a wiki should be created in the future for those interested in introducing dev ops with octobercms in the installation section.

I understand the vision for core, and relying on the plugin versioning :+1 so will continue evaluation from this angl. That being said, things do go wrong, and I imagine some people would still feel more comfortable with a method of installing specific build numbers. How about allow specifying a build number using https://octobercms.com/api/installer/{{build_number}}? Can make a separate PR for this if you want.

At the last resort, vendoring everything into a single repo would seem to be the logical approach left viable. 

Happy to close this.

**Comment by daftspunk** [comment]
Tags have been created for every build from stable onwards (Build 319+)

https://github.com/octobercms/october/releases
https://github.com/octobercms/library/releases

See https://github.com/octobercms/october/issues/2256#issuecomment-266176745 for further details
